Forge.TreeWalker/src/TreeWalkerParameters.cs

```diff
@@ -117,7 +125,7 @@ public class TreeWalkerParameters
         #endregion
 
         /// <summary>
-        /// Instantiates a TreeWalkerParameters object with the properies that are required to instantiate a TreeWalkerSession object.
+        /// Instantiates a TreeWalkerParameters object with the properties that are required to instantiate a TreeWalkerSession object.
```

Comment:
>. [](start = 134, length = 1)

add comment mentioning one should prefer the other ctor to avoid deserializing each time. This ctor is here for backward compatibility

Reply:
I'll put this in future PR, thanks.